src/main/java/de/mediathekview/mserver/crawler/orf/tasks/OrfFilmDetailTask.java

```diff
@@ -0,0 +1,227 @@
+package de.mediathekview.mserver.crawler.orf.tasks;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+import de.mediathekview.mlib.daten.Film;
+import de.mediathekview.mlib.daten.Resolution;
+import de.mediathekview.mserver.base.utils.HtmlDocumentUtils;
+import de.mediathekview.mserver.crawler.basic.AbstractCrawler;
+import de.mediathekview.mserver.crawler.basic.AbstractDocumentTask;
+import de.mediathekview.mserver.crawler.basic.AbstractUrlTask;
+import de.mediathekview.mserver.crawler.orf.OrfEpisodeInfoDTO;
+import de.mediathekview.mserver.crawler.orf.OrfTopicUrlDTO;
+import de.mediathekview.mserver.crawler.orf.OrfVideoInfoDTO;
+import de.mediathekview.mserver.crawler.orf.parser.OrfEpisodeDeserializer;
+import de.mediathekview.mserver.crawler.orf.parser.OrfVideoDetailDeserializer;
+import java.lang.reflect.Type;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import mServer.crawler.CrawlerTool;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jsoup.nodes.Document;
+import org.jsoup.select.Elements;
+
+public class OrfFilmDetailTask extends AbstractDocumentTask<Film, OrfTopicUrlDTO> {
+  
+  private static final Logger LOG = LogManager.getLogger(OrfFilmDetailTask.class);
+  
+  private static final String TITLE_SELECTOR = "h3.video_headline";
+  private static final String BROADCAST_SELECTOR = "div.broadcast_information";
+  private static final String TIME_SELECTOR = BROADCAST_SELECTOR + " > time";
+  private static final String DURATION_SELECTOR = BROADCAST_SELECTOR + " > span.meta_duration";
+  private static final String DESCRIPTION_SELECTOR = "div.details_description";
+  private static final String VIDEO_SELECTOR = "div.jsb_VideoPlaylist";
+  private static final String EPISODE_SELECTOR = "li.jsb_PlaylistItemFullscreen";
+  
+  private static final String ATTRIBUTE_DATETIME = "datetime";
+  private static final String ATTRIBUTE_DATA_JSB = "data-jsb";
+
+  private static final DateTimeFormatter DATE_TIME_FORMATTER =
+    DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+  
+  private static final Type OPTIONAL_VIDEOINFO_TYPE_TOKEN = new TypeToken<Optional<OrfVideoInfoDTO>>() {}.getType();
+  private static final Type OPTIONAL_EPISODEINFO_TYPE_TOKEN = new TypeToken<Optional<OrfEpisodeInfoDTO>>() {}.getType();
+  
+  public OrfFilmDetailTask(final AbstractCrawler aCrawler,
+      final ConcurrentLinkedQueue<OrfTopicUrlDTO> aUrlToCrawlDTOs) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "aUrlToCrawlDTOs" object should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)